<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QPlainTextEdit" name="cs_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string>test</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="ip_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="7">
     <widget class="QTreeWidget" name="auxpref_tree">
      <property name="minimumSize">
       <size>
        <width>143</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>UNION</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>auxpref</string>
       </property>
       <item>
        <property name="text">
         <string>auxpref_value</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>auxpref_u16</string>
       </property>
       <item>
        <property name="text">
         <string>New Subitem</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>auxpref_u8</string>
       </property>
       <item>
        <property name="text">
         <string>New Subitem</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPlainTextEdit" name="ea_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPlainTextEdit" name="segpref_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPlainTextEdit" name="flags_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>cs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="ip">
      <property name="text">
       <string>ip</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="ea">
      <property name="text">
       <string>ea</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="segpref">
      <property name="text">
       <string>segpref</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="insnpref">
      <property name="text">
       <string>insnpref</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="flags">
      <property name="text">
       <string>flags</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPlainTextEdit" name="insnpref_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>itype</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPlainTextEdit" name="itype_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>331</width>
     <height>651</height>
    </rect>
   </property>
   <property name="title">
    <string>Operands</string>
   </property>
   <widget class="QTabWidget" name="operandSelector">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>0</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>5</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>6</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>7</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="optree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>331</width>
      <height>611</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Op</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>n</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>type</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>offb</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>offo</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>flags</string>
     </property>
     <item>
      <property name="text">
       <string>OF_NO_BASE_DISP</string>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OF_OUTER_DISP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PACK_FORM_DEF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OF_NUMBER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OF_SHOW</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>dtype</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>UNION</string>
     </property>
     <item>
      <property name="text">
       <string>reg</string>
      </property>
      <item>
       <property name="text">
        <string>New Subitem</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>phrase</string>
      </property>
      <item>
       <property name="text">
        <string>New Subitem</string>
       </property>
      </item>
     </item>
    </item>
    <item>
     <property name="text">
      <string>value</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>addr</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>SPECFLAGS</string>
     </property>
     <item>
      <property name="text">
       <string>specflag1</string>
      </property>
      <item>
       <property name="text">
        <string>New Subitem</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>specflag2</string>
      </property>
      <item>
       <property name="text">
        <string>New Subitem</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>specflag3</string>
      </property>
      <item>
       <property name="text">
        <string>New Subitem</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>specflag4</string>
      </property>
      <item>
       <property name="text">
        <string>New Subitem</string>
       </property>
      </item>
     </item>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Features</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>331</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLabel" name="CF_USE4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="CF_USE6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="CF_USE5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="CF_USE3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="CF_USE1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="CF_USE2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="CF_CHG1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="CF_CHG2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="CF_CHG3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="CF_CHG4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="CF_CHG5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="CF_CHG6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>CF_USE7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="CF_CHG7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="CF_CHG8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CHG8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="CF_SHFT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_SHFT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="CF_HLL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_HLL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="CF_USE7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="CF_USE8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_USE8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="CF_CALL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_CALL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="CF_STOP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_STOP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="CF_JUMP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CF_JUMP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
